setup.cfg

```diff
@@ -16,4 +16,21 @@ profile = black
 combine_as_imports = True
 
 [tool:pytest]
-addopts = --cov-report= --cov=starlette --cov=tests -rxXs
+addopts =
+  --cov-report=term-missing:skip-covered
+  --cov=starlette
+  --cov=tests
+  -rxXs
+  --strict-config
+  --strict-markers
+  --cov-config tests/.ignore_lifespan
```

Comment:
ok ok last thing (I hope 🤞🏼)

This `. ignore_lifespan` file seems to be obsolete. I'll try clean it up in a separate PR.
```suggestion
```